### ISSUE TITLE
Load object from yaml string?

### ISSUE THREAD

**Issue by zhangzerui20** [fabric8io/kubernetes-client#1712]
How can I get Service object from service yaml string. Something like this:

V1Service service = Yaml.loadAs(yamlString, V1Service.class);

**Comment by walnut-tom** [comment]
use util class io.fabric8.kubernetes.client.utils.Serialization, it support json and yaml, and support parameters too.
V1Service service = Serialization.unmarshal(yamlString, V1Service.class);

**Comment by rohanKanojia** [comment]
I think it can be done like this:
```
Service loadedSvc = client.services().load(getClass)(.getResourceAsStream("/path/to/file")).get()
```

**Comment by sknot-rh** [comment]
Is there any workaround for loading `Template` objects? I am trying to load such an object with the `Integer` placeholder however, it is failing. Minimal reproducer:

Run this test in `io.fabric8.openshift.client.server.mock.LoadAsTemplateTest`
```
  @Test
  void unmarshallTemplateObjectWithIntegerPlaceholder() {
    Template template = Serialization.unmarshal(getClass().getResourceAsStream("/template-with-json-params.yml"), Template.class);
  }
```
It fails with `Cannot deserialize value of type `java.lang.Integer` from String "${{CONTAINER_PORT}}": not a valid Integer value` as the validator expects `Integer` value but the placeholder `String` is provided.

**Comment by shawkins** [comment]
@sknot-rh can you move this comment onto #3460

**Comment by shawkins** [comment]
> it should go without saying that this is at least un-expected. Is there an actual issue where this is tracked and we are not aware of? Thank you.

See #4574

**Comment by manusa** [comment]
This was a change of behavior added in 6.4.0.

We did include a breaking change note:

https://github.com/fabric8io/kubernetes-client/blob/c05ac82eff1f3faf5f8353ed3022133696c2683e/CHANGELOG.md#L294-L301

>  * Fix `#4574`: fromServer has been deprecated - it no longer needs to be called.  All get() operations will fetch the resource(s) from the api server.  If you need the context item that was passed in from a resource, load, or resourceList methods, use the item or items method. 

However, there were many changes in that version in it didn't stick out very much.

Many users (specially newer ones) were already assuming that this was the expected behavior. That's the main reason we opted to change it and deprecate-remove `fromServer`.
